docs/leap_sapi/sapi_rest.rst

```diff
@@ -163,7 +163,45 @@ resources and provides example usages.
 Examples Setup
 --------------
 
-The examples below use the following setup:
+To run the examples in this topic, set up your development environment, such as
+iPython, as described on the following tabs.
+
+The ``Accept`` request header is optional; if included, it must be one of the
+following:
+
+============================================ ====== ===================================================================
+Resource                                     Method Accept Request Header
+============================================ ====== ===================================================================
+/bqm/multipart                               POST   application/vnd.dwave.sapi.multipart-initiate+json; version=<x>
+/bqm/multipart/<problem_data_id>/part/<part> PUT    application/vnd.dwave.sapi.multipart-upload+json; version=<x>
+/bqm/multipart/<problem_data_id>/combine     POST   application/vnd.dwave.sapi.multipart-combine+json; version=<x>
+/bqm/multipart/<problem_data_id>/status      GET    application/vnd.dwave.sapi.multipart-status+json; version=<x>
+/problems/                                   POST   application/vnd.dwave.sapi.problems+json; version=<x>
+/problems/                                   DELETE application/vnd.dwave.sapi.problems+json; version=<x>
+/problems/<problem_id>                       DELETE application/vnd.dwave.sapi.problem-data+json; version=<x>
+/problems/                                   GET    application/vnd.dwave.sapi.problems+json; version=<x>
+/problems/<problem_id>/                      GET    application/vnd.dwave.sapi.problem+json; version=<x>
+/problems/<problem_id>/info                  GET    application/vnd.dwave.sapi.problem-data+json; version=<x>
+/problems/<problem_id>/answer/               GET    application/vnd.dwave.sapi.problem-answer+json; version=<x>
+/problems/<problem_id>/messages/             GET    application/vnd.dwave.sapi.problem-message+json; version=<x>
+/solvers/remote/                             GET    application/vnd.dwave.sapi.solver-definition-list+json; version=<x>
+/solvers/remote/<solver_id>/                 GET    application/vnd.dwave.sapi.solver-definition+json; version=<x> [#]_
+/solvers/remote/<solver_name>                GET    application/vnd.dwave.sapi.solver-definition+json; version=<x> [#]_
+============================================ ====== ===================================================================
+
+``<x>`` is the version of the Accept request-header; for example, ``3.0.0``.
+
+.. [#]
+    If ``version=3.0.0`` is specified, the ``identity``
```

Comment:
I wouldn't be so specific in the docs, saying it has to be `3.0.0`. Both `3` and `3.0` work, so I would use `version=3`.

Also, it's important to note that this is not the API version per se, rather, it's the resource representation version that client prefers. It's at server's discretion to respond with the best representation. For example, if client asks for v3.1, the server might respond with 3.0.0 (if 3.1 is not available). But also, if it asks for 2.0 or 4.0 after the deprecation period, the server will respond with the latest v3 available.

Reply:
also note: only some of endpoints have new version (the ones affected)
so instead of including a list, maybe better to include the value under each endpoint?